plugin-infra/go-plugin-access/src/com/thoughtworks/go/plugin/access/authorization/AuthorizationMessageConverterV1.java

```diff
@@ -168,6 +165,46 @@ public String getProcessRoleConfigsResponseBody(List<PluginRoleConfig> roles) {
         return GSON.toJson(list);
     }
 
+    @Override
+    public String grantAccessRequestBody(List<SecurityAuthConfig> authConfigs) {
+        Map<String, Object> requestMap = new HashMap<>();
+        requestMap.put("auth_configs", getAuthConfigs(authConfigs));
+
+        return GSON.toJson(requestMap);
+    }
+
+    @Override
+    public Map<String, String> getCredentials(String responseBody) {
+        Type type = new TypeToken<Map<String, String>>() {
+        }.getType();
+
+        return GSON.fromJson(responseBody, type);
+    }
+
+    @Override
+    public String authenticateUserRequestBody(Map<String, String> credentials, List<SecurityAuthConfig> authConfigs, List<PluginRoleConfig> roleConfigs) {
+        Map<String, Object> requestMap = new HashMap<>();
+
+        requestMap.put("credentials", credentials);
+        requestMap.put("auth_configs", getAuthConfigs(authConfigs));
+        requestMap.put("role_configs", getRoleConfigs(roleConfigs));
+        return GSON.toJson(requestMap);
+    }
+
+    @Override
```

Comment:
Rename method to `authorizationServerRedirectUrl` and contract to `authorization_server_redirect_url` instead of `identity_provider_url `?